docs/Remote-zqd.md

```diff
@@ -0,0 +1,310 @@
+sample
+  id:           sp_1iIUQUsAAhUw7kpF0UoOcFNWxqj
+  name:         sample
+  data_path:    file:///home/ubuntu/.config/Brim/data/spaces/sp_1iIUQUsAAhUw7kpF0UoOcFNWxqj
+  storage_kind: filestore
+  span:         2018-03-24T17:15:20Z+21m9.558041001s
+  size:         80.05MB
+  pcap_support: false
+  pcap_size:    0B
+  pcap_path:    
+  parent_id:    
+wrccdc
+  id:           sp_1iIUcu5BQV92uz7v6rbIqSzT4BI
+  name:         wrccdc
+  data_path:    file:///home/ubuntu/.config/Brim/data/spaces/sp_1iIUcu5BQV92uz7v6rbIqSzT4BI
+  storage_kind: filestore
+  span:         2018-03-23T19:58:22Z+1m51.694792001s
+  size:         18.52MB
+  pcap_support: true
+  pcap_size:    500.0MB
+  pcap_path:    file:///home/ubuntu/wrccdc.2018-03-23.010014000000000.pcap
+  parent_id:    
+```
+
+# Accessing Our Remote `zqd`
+
+Now that we've got data imported into our remote `zqd`, we'll access it from
+the Brim app that's running on our Mac laptop.
+
+The option to initiate a remote connection is available by clicking the
+pull-down that's normally above the Spaces list in the left panel.
+
+![New Connection option](media/New-Connection-option.png)
+
+A window will pop up into which we can specify a name for our remote connection
+along with its hostname or IP address. An optional port specification may also
+be included if it's listening on a port other than the default `9867`.
+
+![New Connection window](media/New-Connection-window.png)
+
+Now the Spaces on our remote `zqd` will appear in the left panel just as we're
+accustomed to seeing when working with local data. We can now enter ZQL queries
+and perform all normal workflows. For our Space that was based on the imported
+packet capture, we can extract flows into Wireshark via the **Packets** button
+as usual.
+
+![Opening a remote flow](media/Remote-Flow-Wireshark.png)
+
+You can import logs (but not pcaps) directly from your Brim app to the remote
+`zqd` in the same manner as you've been doing it locally.
+
+![Importing logs to remote zqd](media/Remote-Log-Import.gif)
+
+If you were to attempt to import pcaps directly to the remote `zqd`, you'd see
+an error pop up like the one shown below. This is due to the fact that pcap
+import is currently implemented by posting the local file path from Brim to
+`zqd`, so the import fails when the remote `zqd` doesn't find the pcap at that
+at that path on its filesystem.
+
+![Failing pcap import to remote zqd](media/Remote-pcap-Import-Fail.gif)
+
+While new Spaces you create by importing logs via Brim will show up
+immediately in your app, additional Spaces created on the remote end via `zapi`
+or by other users importing to the same remote `zqd` will not appear
+automatically in your Brim app. To trigger an immediate refresh, select
+**View > Reload** from the Brim drop-down menu.
+
+![Reload to view new Spaces](media/Reload-New-Spaces.gif)
+
+A remote connection can be deleted by selecting the **Get Info** option in the
+pull-down and clicking **Remove**.
```

Comment:
It might be good to use the word "removed" instead of "deleted" here. Or add some words saying that the connection is only removed from the app. The remote zqd is not shutdown nor the data directory deleted.

Reply:
Thanks @jameskerr. Great suggestion. I've pushed a change to clarify this.